src/pages-conditional/what-is-ethereum.js

```diff
@@ -216,101 +219,105 @@ const StyledCallout = styled(Callout)`
-      alt: "The symbol for Ether (ETH)",
+      alt: "what-ethereum-native-alt",
       image: data.eth.childImageSharp.fixed,
-      description:
-        "Ethereum's native cryptocurrency and equivalent to Bitcoin. You can use ETH on Ethereum applications or for sending value to friends and family. ",
+      description: "whats-ethereum-native-crypto",
     },
     {
-      title: "Wallets",
+      title: "whats-ethereum-wallets",
       to: "/wallets/",
-      alt:
-        "An illustration of a robot with a safe for a torso, used to represent Ethereum wallets",
+      alt: "whats-ethereum-native-img-alt",
       image: data.wallets.childImageSharp.fixed,
 
-      description:
-        "How you manage your ETH and your Ethereum account. You'll need a wallet to get started – we'll help you choose one.",
+      description: "whats-etheruem-wallets-desc",
     },
     {
-      title: "Ethereum dapps",
+      title: "whats-ethereum-dapps-title",
       to: "/dapps/",
-      alt:
-        "An illustration of a doge using an Ethereum application on a computer",
+      alt: "whats-ethereum-dapps-img-alt",
       image: data.dapps.childImageSharp.fixed,
-      description:
-        "Products and services that run on Ethereum. There are dapps for finance, work, social media, gaming and more – meet the apps for our digital future.",
+      description: "whats-ethereum-dapps-desc",
     },
   ]
   return (
     <Page>
       <PageMetadata
-        title="What is Ethereum?"
-        description="Learn about Ethereum, what it does and how to try it for yourself."
+        title={intl.formatMessage({
+          id: "whats-ethereum-meta-title",
+          defaultMessage: getDefaultMessage("whats-ethereum-meta-title"),
+        })}
+        description={intl.formatMessage({
+          id: "what-ethereum-meta-description",
+          defaultMessage: getDefaultMessage("what-ethereum-meta-description"),
+        })}
         image={data.ogImage.childImageSharp.fixed.src}
       />
       <HeroContent>
         <HeroContainer>
           <Header>
-            <Title>What is Ethereum?</Title>
-            <Slogan>The foundation for our digital future</Slogan>
-            <Subtitle>Ethereum is open to everyone. </Subtitle>
-            <SubtitleTwo> All you need is a wallet to take part.</SubtitleTwo>
+            <Title>
+              <Translation id="whats-ethereum-title" />
+            </Title>
+            <Slogan>
+              <Translation id="whats-ethereum-desc" />
+            </Slogan>
+            <Subtitle>
+              <Translation id="whats-ethereum-accessibility" />
+            </Subtitle>
+            <SubtitleTwo>
+              <Translation id="whats-ethereum-tools-needed" />
+            </SubtitleTwo>
           </Header>
           <Hero
             fluid={data.hero.childImageSharp.fluid}
-            alt="Illustration of a person peering into a bazaar, meant to represent Ethereum"
+            alt="whats-ethereum-alt-img-bazaar"
```

Comment:
Since, you're not using the `<Translation/>` component here, you'll need to use the `formatMessage` function - see here for an example: https://github.com/ethereum/ethereum-org-website/blob/dev/src/pages/index.js#L373